server/db/models/index.js

```diff
@@ -1,11 +1,18 @@
 const User = require('./user')
+const Fruit = require('./fruit')
+const Order = require('./order')
 
 /**
  * If we had any associations to make, this would be a great place to put them!
  * ex. if we had another model called BlogPost, we might say:
  *
  *    BlogPost.belongsTo(User)
  */
+User.hasMany(Order)
+Order.belongsTo(User)
+
+Order.hasMany(Fruit)
+Fruit.hasMany(Order)
 
```

Comment:
Association needed for Cart::

User.belongsToMany(Fruit, {through: 'cart'})
Fruit.belongsToMany(User, {through: 'cart'})